src/main/java/org/jenkinsci/plugins/runselector/selectors/SpecificRunSelector.java

```diff
@@ -95,8 +95,13 @@ public String getBuildNumber() {
         return run;
     }
 
-    @Extension(ordinal=-10)
-    public static final Descriptor<RunSelector> DESCRIPTOR =
-            new SimpleRunSelectorDescriptor(
-                SpecificRunSelector.class, org.jenkinsci.plugins.runselector.Messages._SpecificRunSelector_DisplayName());
+    @Symbol("specificRun")
```

Comment:
This is related to my comment on the discussion page: I would prefer to see these two symbols: `buildNumber` and `permalink`, rather than `specific`. I don't find `specific` to be as self-explanatory as having two different symbols. However I do not have a strong opinion. If you think it adds value you can change, if not just ignore this comment.